code/src/main/java/com/onlinecontacttracing/messaging/GeneratedUserId.java

```diff
@@ -0,0 +1,20 @@
+package com.onlinecontacttracing.messsaging;
+
+import java.util.UUID;
+
+/**
+ * Generates a user ID for users who have not logged in in the form of anon-*
+ */
+public class GeneratedUserId {
+  private static String userId;
+  private static UUID uuid = new UUID(123456789, 987654321);
```

Comment:
So this is an anonymous user?

Also, I do not believe these fields should be static. Static means all calls to new GenerateUserId() and the objects returned will point at the same piece of memory.

Basically, only user static for constant values. There are rare occasions where you use it for mutable values, but you will not run into them often.

Reply:
Yes this is the anonymous user, I'll remove static.